recipes/mongodb_org_repo.rb

```diff
@@ -38,7 +38,7 @@
 when 'rhel', 'fedora'
   yum_repository 'mongodb' do
     description 'mongodb RPM Repository'
-    baseurl "http://downloads-distro.mongodb.org/repo/redhat/os/#{node['kernel']['machine']  =~ /x86_64/ ? 'x86_64' : 'i686'}"
+    baseurl "#{node['mongodb']['repo']}/#{node['kernel']['machine']  =~ /x86_64/ ? 'x86_64' : 'i686'}"
```

Comment:
most of the cookbook uses strings. I don't particularly have a preference, but if either of you ever wants to go through and make things all the same, I'll :joy: